model/project_parser.go

```diff
@@ -367,6 +367,30 @@ func (pbv *parserBV) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+// checks that all fields are empty besides name and tasks
```

Comment:
Nit: comments should be sentences (i.e. capital letters, punctuation); linebreaks shouldn't be considered sentence breaks, since this isn't true for longer sentences. Also, while this isn't as important for private functions, it is conventional for function headers to start with the function name "canMerge checks that ..."